### PR TITLE
DOC-10130 Errors on Sync Gateway Supported Environments Page

### PR DESCRIPTION
**Pull request by RayOffiah**
https://issues.couchbase.com/browse/DOC-10130

Fixed

### CODE REVIEW

File: modules/ROOT/pages/_partials/sgw-supported-os.adoc [hunk 1]
```diff
@@ -26,8 +26,7 @@
 a| 2019 +
 2016 (64-bit) -- deprecated at {version-full}
 
-| Apple M1 ARM64
-| macOS 11
+
```
Comment (djpongh) [inline modules/ROOT/pages/_partials/sgw-supported-os.adoc:29]: this should be moved down to table 3 (need to explicitly call out Arm64 )

**Review by djpongh — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/_partials/sgw-supported-os.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/ROOT/pages/_partials/sgw-supported-os.adoc:29]: Fixed

**Comment by RayOffiah** [comment]
@djpongh 

Could you take another look at the changes? 

The page doesn't usually mention the hardware under the operating system, so it does stand out a bit.

Thanks.

**Comment by djpongh** [comment]
looks good - Apple is a special case because their old systems are x86 based and the new systems are ARM64 based

**Review by djpongh — APPROVED** [review]

**Pull request merged**